lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml

```diff
@@ -83,39 +75,63 @@
     - 'echo "Pin: origin packages.magma.etagecom.io" >> /etc/apt/preferences.d/magma-preferences'
     - 'echo "Pin-Priority: 900" >> /etc/apt/preferences.d/magma-preferences'
 
+- name: Installing magma.
+  become: yes
+  apt:
+    name: "{{ packages }}"
+  vars:
+    packages:
+      - magma
+
 - name: Start service openvswitch-switch.
   become: yes
   service:
     name: openvswitch-switch
     state: started
+  tags:
+    - skipfirstinstall
```

Comment:
It looks like baremetal provisioning is the only place where the ovs_deploy role is used, why keep these tasks around if we're going to skip them?

Reply:
I had plan to recycle that role but I can just remove it for now.